### PR TITLE
Sign the team roster

### PR DESCRIPTION
**Pull request by idrysdale**

### CODE REVIEW

File: team/team.go [hunk 1]
```diff
@@ -58,9 +60,20 @@ func Save(team Team, fluidkeysDirectory string) error {
...
+
+	signature, err := signingKey.MakeArmoredDetachedSignature(roster.Bytes())
+	if err != nil {
+		return fmt.Errorf("failed to sign team roster: %v", err)
+	}
+
 	if err = atomic.WriteFile(rosterFilename, roster); err != nil {
 		return fmt.Errorf("failed write team roster: %v", err)
 	}
+	err = atomic.WriteFile(signatureFilename, bytes.NewBufferString(signature))
```
Comment (paulfurley) [inline team/team.go]: Would `[]byte(signature)` work instead of `bytes.NewBufferString(signature)` ?

File: team/team_test.go [hunk 2]
```diff
@@ -173,9 +183,34 @@ func TestSave(t *testing.T) {
...
+				t.Fatalf("couldn't read " + rosterFilepath)
+			}
+
+			signatureFilepath := filepath.Join(teamSubdir, "roster.toml.asc")
+			signature, err := ioutil.ReadFile(signatureFilepath)
+			if err != nil {
+				t.Fatalf("couldn't read " + signatureFilepath)
+			}
+
+			verifyRosterSignature(t, roster, signature, signingKey)
```
Comment (paulfurley) [inline team/team_test.go]: nice!!

**Comment by paulfurley** [comment]
:heart:

File: team/team.go (hunk 1 above)
Comment (idrysdale) [reply team/team.go]: Sadly not...
```
cannot use ([]byte)(signature) (type []byte) as type io.Reader in argument to "github.com/fluidkeys/fluidkeys/vendor/github.com/natefinch/atomic".WriteFile:
	[]byte does not implement io.Reader (missing Read method)
```

**Pull request merged**